javascript/ql/src/experimental/Security/CWE-942/CorsPermissiveConfiguration.ql

```diff
@@ -0,0 +1,20 @@
+/**
+ * @name overly CORS configuration
+ * @description Misconfiguration of CORS HTTP headers allows CSRF attacks.
+ * @kind path-problem
+ * @problem.severity error
+ * @security-severity 7.5
+ * @precision high
+ * @id js/cors-misconfiguration
+ * @tags security
+ *       external/cwe/cwe-942
+ */
+
+import javascript
+import semmle.javascript.security.dataflow.CorsPermissiveConfigurationQuery
```

Comment:
This import doesn't work after you moved to experimental. 

I think you should just move the two `.qll` into the same folder as this `.ql` file.  
That fits well for an experimental query.